mailpoet/package.json

```diff
@@ -137,7 +137,7 @@
     "@types/js-cookie": "^3.0.3",
     "@types/lodash": "^4.14.196",
     "@types/mocha": "^10.0.1",
-    "@types/node": "^18.17.1",
+    "@types/node": "^20.4.5",
```

Comment:
It seems that currently, we're using Node.js version 19.7.0. We should either upgrade to version 20 or keep the types in sync.